lynx/patterns/link-in-bio-purple-background.php

```diff
@@ -0,0 +1,86 @@
+<div class="wp-block-group has-text-color has-background has-link-color" style="border-radius:100px;color:#111111;background-color:#ffffffe6;margin-top:10px;margin-bottom:10px;padding-top:10px;padding-right:10px;padding-bottom:10px;padding-left:10px"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+<div class="wp-block-group"><!-- wp:social-links {"customIconColor":"#111111","iconColorValue":"#111111","className":"is-style-logos-only"} -->
+<ul class="wp-block-social-links has-icon-color is-style-logos-only"><!-- wp:social-link {"url":"#","service":"wordpress"} /--></ul>
+<!-- /wp:social-links -->
+
+<!-- wp:paragraph -->
+<p><a href="#"><?php echo  esc_html__( 'Blog', 'lynx' ); ?></a></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"color":{"background":"#ffffffe6","text":"#111111"},"border":{"radius":"100px"},"spacing":{"blockGap":"0px","padding":{"top":"10px","right":"10px","bottom":"10px","left":"10px"},"margin":{"top":"10px","bottom":"10px"}},"elements":{"link":{"color":{"text":"#111111"}}}}} -->
+<div class="wp-block-group has-text-color has-background has-link-color" style="border-radius:100px;color:#111111;background-color:#ffffffe6;margin-top:10px;margin-bottom:10px;padding-top:10px;padding-right:10px;padding-bottom:10px;padding-left:10px"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+<div class="wp-block-group"><!-- wp:social-links {"className":"is-style-default"} -->
+<ul class="wp-block-social-links is-style-default"><!-- wp:social-link {"url":"#","service":"tumblr"} /--></ul>
+<!-- /wp:social-links -->
+
+<!-- wp:paragraph -->
+<p><a href="#"><?php echo  esc_html__( 'Tumblr', 'lynx' ); ?></a></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"color":{"background":"#ffffffe6","text":"#111111"},"border":{"radius":"100px"},"spacing":{"blockGap":"0px","padding":{"top":"10px","right":"10px","bottom":"10px","left":"10px"},"margin":{"top":"10px","bottom":"10px"}},"elements":{"link":{"color":{"text":"#111111"}}}}} -->
+<div class="wp-block-group has-text-color has-background has-link-color" style="border-radius:100px;color:#111111;background-color:#ffffffe6;margin-top:10px;margin-bottom:10px;padding-top:10px;padding-right:10px;padding-bottom:10px;padding-left:10px"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+<div class="wp-block-group"><!-- wp:social-links {"className":"is-style-logos-only"} -->
+<ul class="wp-block-social-links is-style-logos-only"><!-- wp:social-link {"url":"#","service":"twitter"} /--></ul>
+<!-- /wp:social-links -->
+
+<!-- wp:paragraph -->
+<p><a href="#"><?php echo  esc_html__( 'Twitter', 'lynx' ); ?></a></p>
```

Comment:
Do we really need to translate service names?

Reply:
If the theme is to be submitted to the .org themes directory, I believe there is still the requirement that all strings be escaped + translated. 

If it's not going to be submitted there, then I don't think it's necessary to translate, but also not sure why it would be added to this repo instead of just living on dotcom.

Reply:
We won't be submitting Lynx to .org. I added it here because I thought all free themes are worked on in this GH repo, not just the ones going to .org. Is this ok to keep it here?

I'll take care of removing the translation around social media services, but likely after this initial commit is merged.

Reply:
>  I thought all free themes are worked on in this GH repo, not just the ones going to .org

I don't think this would be the only theme on wpcom but not in themes directory.  I can't recall which but I think there are others.  For workflow I think all free themes being here (except those that are retired or near retirement) makes sense.